Xamarin.Essentials/Connectivity/Connectivity.android.cs

```diff
@@ -11,22 +11,46 @@ namespace Xamarin.Essentials
     public partial class Connectivity
     {
         static ConnectivityBroadcastReceiver conectivityReceiver;
+        static Intent connectivityIntent = new Intent(Platform.EssentialsConnectivityChanged);
```

Comment:
Is this safe to reuse? I think it is mostly immutable, but any other info on this?

Reply:
hmmmm we could probably create a new Intent each time....?